### DISCUSSION TITLE
How do I use the subcommand tune when using the CLI?

### DISCUSSION THREAD

**Discussion by goncalomcorreia** [Lightning-AI/pytorch-lightning#11028] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
When I run:

```
python main.py tune -h
```

At the end of the output, we can read:

```
Runs routines to tune hyperparameters before training:
  --scale_batch_size_kwargs SCALE_BATCH_SIZE_KWARGS
                        Arguments for :func:`~pytorch_lightning.tuner.batch_size_scaling.scale_batch_size` (type: Union[Dict[str, Any], null], default: null)
  --lr_find_kwargs LR_FIND_KWARGS
                        Arguments for :func:`~pytorch_lightning.tuner.lr_finder.lr_find` (type: Union[Dict[str, Any], null], default: null)
```

I would like to run the learning rate finder. From the [documentation](https://pytorch-lightning.readthedocs.io/en/latest/advanced/lr_finder.html), it seems `lr_find` takes as input a trainer and a model. How do I get those from the config?

I tried something like this and it didn't work:
```
python main.py tune --lr_find_kwargs {"model": mypackage.models.MyVAE}
```

Can someone share a working example of using the `tune` subcommand with the CLI?
